Cleanse/BinderArities.swift

```diff
@@ -22,15 +22,15 @@ extension BindToable {
 /// - parameter factory: Takes arguments required to construct `Element` passed to the `bind()` function
 ///
 /// - Note: This method was auto-generated by CleanseGen/main.swift
-@discardableResult public func to<P_1, P_2>(file: StaticString=#file, line: Int=#line, function: StaticString=#function, factory: @escaping (P_1, P_2) -> Input)  -> BindingReceipt<Input> {
+@discardableResult public func to<P_1, P_2>(file: StaticString=#file, line: Int=#line, function: StaticString=#function, factory: @escaping ((P_1, P_2)) -> Input)  -> BindingReceipt<Input> {
```

Comment:
Might be worth adding a comment about why embedding in another tuple fixes Swift 5 compatibility